components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -736,12 +736,16 @@
             var collaboratorsCount = formData.collaborator ? (formData.collaborator.size &amp;&amp; formData.collaborator.size() || 1) : 0;
             _this._updateDisplayedCollaborators(formData.collaborator);
             if (collaborators) {
-              var topCollaborator = $$('#manage-collaborators tr').map(function(item) {return item.down('select').selectedIndex + ':' + item.down('td:nth-child(2)').innerText}).max();
-              topCollaborator = topCollaborator.substring(topCollaborator.indexOf(':') + 1);
-              var collaboratorText = topCollaborator;
-              if (collaboratorsCount &gt; 1) {
+              if (collaboratorsCount == 0) {
+                collaboratorText = "$services.localization.render('phenotips.patientAccessRightsManagement.collaboratorsSummary.noCollaborators')";
+              } else {
                 --collaboratorsCount;
-                collaboratorText = collaboratorsCount == 1 ? "$services.localization.render('phenotips.patientAccessRightsManagement.collaboratorsSummary.oneCollaborator', ['" + collaboratorText + "', '" + collaboratorsCount + "'])" : "$services.localization.render('phenotips.patientAccessRightsManagement.collaboratorsSummary.xCollaborators1', ['" + collaboratorText + "', '" + collaboratorsCount + "'])";
+                var topCollaborator = $$('#manage-collaborators tr').map(function(item) {return item.down('select').selectedIndex + ':' + item.down('td:nth-child(2)').innerText}).max();
+                topCollaborator = topCollaborator.substring(topCollaborator.indexOf(':') + 1);
+                var collaboratorText = topCollaborator;
+                if (collaboratorsCount &gt; 0) {
+                  collaboratorText = collaboratorsCount == 1 ? "$services.localization.render('phenotips.patientAccessRightsManagement.collaboratorsSummary.oneCollaborator', ['" + collaboratorText + "', '" + collaboratorsCount + "'])" : "$services.localization.render('phenotips.patientAccessRightsManagement.collaboratorsSummary.xCollaborators', ['" + collaboratorText + "', '" + collaboratorsCount + "'])";
```

Comment:
I think it's better to use ? for shorter expressions and put an explicit if here.